crypto/initthread.c

```diff
@@ -267,7 +267,7 @@ void *ossl_thread_event_ctx_new(OSSL_LIB_CTX *libctx)
         return NULL;
 
     if (!CRYPTO_THREAD_init_local(tlocal, NULL)) {
-        goto err;
+        goto deinit;
```

Comment:
The style checker is catching this.  I know this didn't introduce it, but coding style says that, if you have a conditional with only one statement, you don't put curly braces around it.  Removing the curly braces here will get the style check ci job to pass